meal-planner-fredericton-ui/src/dashboard/Dashboard.jsx

```diff
@@ -153,6 +154,36 @@ function doDeleteMeapPlan({ id }) {
   });
 }
 
+const DeleteDialog = ({mealPlan, onDelete, onCancel}) => {
```

Comment:
Created a new React component DeleteDialog so that it can have mealplan (selected row) as one of the props.